docs/conmon.8.md

```diff
@@ -0,0 +1,144 @@
+conmon 8 "User Commands"
+==================================================
+
+# NAME
+
+conmon - container monitor utility
+
+# SYNOPSIS
+
+conmon [options]
+
+# DESCRIPTION
+
+conmon is a command line program for monitorying and managing the lifecycle of
+Linux containers that follow the Open Container Initiative (OCI) format.
+
+# APPLICATION OPTIONS
+
+**--api-version**
+Conmon API version to use.
+
+**-b**, **--bundle**
+Location of the OCI Bundle path.
+
+**-c**, **--cid**
+Identification of Container.
+
+**--exec-attach**
+Attach to an exec session.
+
+**-e**, **--exec**
+Exec a command into a running container.
+
+**--exec-process-spec**
+Path to the process spec for execution.
+
+**--exit-command**
+Path to the program to execute when the container terminates its execution.
+
+**--exit-command-arg**
+Additional arg to pass to the exit command.  Can be specified multiple time.
+
+**--exit-delay**
+Delay before invoking the exit command (in seconds).
+
+**--exit-dir**
+Path to the directory where exit files are written.
+
+**-h**, **--help**
+Show help options.
+
+**-i**, **--stdin**
+Create !terminal stdin pipe.
+
+Create a "fake" main fd so that we can use the same epoll code in both cases.
+The workerfd_*s will be closed after we dup over everything. We use pipes here because
+open(/dev/std{out,err}) will fail if we used anything else (and it wouldn't be a good idea to
+create a new pty pair in the host).
```

Comment:
I'd drop the conversational tone and the we/you use in this section.

Reply:
Re-reading, this all sounds like a good code comment, not a description of the functionality from a user perspective.

Reply:
Yes Want @haircommander to rewrite this section for Human consumption.

Reply:
I think we can just drop this whole section

Reply:
Removed